app/db/model.py

```diff
@@ -59,6 +69,7 @@ class Job(Base):
     group_id: Mapped[UUID | None]
     vlab_id: Mapped[UUID] = mapped_column(ForeignKey("account.id"), index=True)
     proj_id: Mapped[UUID] = mapped_column(ForeignKey("account.id"), index=True)
+    name: Mapped[str | None] = mapped_column(String(255), nullable=True)
```

Comment:
I don't think there is an advantage in setting a specific max length. Is there a reason? Also, nullable is not required because it's implied by the type annotation.

```suggestion
    name: Mapped[str | None]
```

Reply:
Thank you Gianluca, I've applied your suggestion. Probably will add validation on the API side a bit later.

Reply:
Ah, need to remove unnecessary imports

Reply:
Ok, should be fixed now.